cmd/geoipupdate/config.go

```diff
@@ -141,6 +147,11 @@ func NewConfig(
 	}
 	config.Proxy = proxyURL
 
+	if (config.AccountID == 0 || config.AccountID == 999999) && config.LicenseKey == "000000000000" {
```

Comment:
A comment wouldn't hurt so that we remember why we have these special values, e.g., https://github.com/maxmind/geoipupdate/blob/master/bin/geoipupdate.c#L317-L320